nexus/src/external_api/views.rs

```diff
@@ -421,11 +421,12 @@ impl From<model::Sled> for Sled {
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
 pub struct User {
     pub id: Uuid,
+    pub external_id: String,
```

Comment:
It just occurred to me it might makes sense to call this `display_name`. The fact that it comes from `external_id` is an implementation detail, and when we change how it works, the external API wouldn't have to change.